crates/ethereum-forks/src/hardfork/ethereum.rs

```diff
@@ -380,7 +380,7 @@ impl EthereumHardfork {
             (
                 Self::Paris,
                 ForkCondition::TTD {
-                    activation_block_number: 1735371,
+                    activation_block_number: 1450409,
                     fork_block: Some(1735371),
```

Comment:
smol issue introduced in #13362 where the exclusive sepolia fork_block was incorrectly taken as the activation block here -.-